src/Analyser/IgnoredError.php

```diff
@@ -48,10 +48,18 @@ public static function shouldIgnore(
 		?string $path
 	): bool
 	{
+		// normalize newlines to allow working with ignore-patterns independent of used OS newline-format
+		$errorMessage = $error->getMessage();
+		$errorMessage = preg_replace('/\r\n|\r|\n/', "\n", $errorMessage);
```

Comment:
wdyt about moving this message-normalization into a single spot? does a class exists in which this would fit in well?

Reply:
It's fine like this :)